ci/gpu/build.sh

```diff
@@ -42,7 +42,7 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
+conda install "rmm=$MINOR_VERSION.*" "cmake_setuptools" "cudatoolkit=$CUDA_REL" \
```

Comment:
`cmake_setuptools` is already included in the `gpuci-build-environment`. https://github.com/rapidsai/gpuci-build-environment/blob/master/Dockerfile#L87
```suggestion
conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
```